scanner/src/main/kotlin/experimental/ExperimentalScanner.kt

```diff
@@ -223,6 +223,9 @@ class ExperimentalScanner(
                             )
                         )
                     }
+
+                    // If resolving the nested repositories failed, we can still scan the main repository.
+                    controller.addNestedProvenance(provenance, NestedProvenance(provenance, emptyMap()))
```

Comment:
I'm unsure whether this dummy entry is the right approach, or whether we rather should change the linked lookup in `ScanController` to handle non-existing entries. Maybe again @mnonnenmacher can advise?